js/plugins/controlPanel.js

```diff
@@ -271,22 +289,52 @@ module.exports = function () {
 
             if (browser.tracksLibrary && browser.tracksLibrary.length) {
               var tracksLibrary = $.map(browser.tracksLibrary, function (track) {
-                return track.prototype.name && track.prototype.removable !== false ? [[ track.prototype.name.toLowerCase(), track ]] : undefined;
+                var name = typeof(track) === 'object' ? track.name : track.prototype.name;
```

Comment:
Could be worth moving `typeof(track) === 'object'` into a const, as it's a repeated check. Not sure of the best name for it though?